lib/rjgit_adapter/git_layer_rjgit.rb

```diff
@@ -15,6 +15,11 @@ module Git
 
     import 'org.eclipse.jgit.revwalk.RevWalk'
     import 'org.eclipse.jgit.lib.ObjectId'
+    import org.eclipse.jgit.lib.ConfigConstants
+
+    def self.global_default_branch
+      org.eclipse.jgit.util.SystemReader.getInstance().getUserConfig().getString(ConfigConstants::CONFIG_INIT_SECTION, nil, ConfigConstants::CONFIG_KEY_DEFAULT_BRANCH)
+    end
```

Comment:
This oneliner is not very pretty, but I wonder if it's worth abstracting this in RJGit. For clarity, this is necessary because `RJGit::Configuration` only gives access to repo-specific config.

Reply:
I agree it makes sense to provide an interface. I'll open an issue on RJGit but leave this as is for now.